### DISCUSSION TITLE
Error showing me on Build With Jekyll

### DISCUSSION THREAD

**Discussion by javiyadeep** [community/community#32137] [Other Feature Feedback, Questions, & Ideas]
Hello,

I am facing some issues to deploy my project,
error showing me on Build With Jekyll,

Liquid Exception: Liquid syntax error (line 35): Variable '{{!-- ## owl.lazyload.js LazyLoad require `class="owl-lazy"` and link to image inside data: data-src="url_to_img" or/and data-src-retina="url_to_highres_img". There is also option to load images into background but this needs to be tested. Usage: ``` $('.owl-carousel').owlCarousel({ lazyLoad:true }' was not properly terminated with regexp: /\}\}/ 

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/71928974/188655483-0a860b2c-4a2c-4685-866e-5865ce656405.png">

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/71928974/188655812-74c77ce3-bf29-47b6-9359-81a00529d853.png">

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/71928974/188655363-88fb8050-9591-49a0-be1f-f5a98e744353.png">

**Comment by yoannchaudet** [comment]
Your site does not look like a Jekyll one. Pages build sites with Jekyll automatically when not told to do anything else. You may want to add a `.nojekyll` file at the root of your Pages source branch. Or better, maybe use a custom Actions workflow to build your Node site? (I see a `node_modules` folder :)).
